### PR TITLE
adding documentation for 'classpathExtensions'

### PR DESCRIPTION
**Pull request by simschla**
as discussed in PR https://github.com/flyway/flyway/pull/1738

### CODE REVIEW

File: documentation/gradle/index.md [hunk 1]
```diff
@@ -140,6 +140,33 @@ task migrateDatabase2(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
...
+    schemas = ['schema1', 'schema2', 'schema3']
+    placeholders = [
+        'keyABC': 'valueXYZ',
+        'otherplaceholder': 'value123'
+    ]
+    classpathExtensions = [ configurations.migration ] //your custom extension(s) of classpath to scan
+
+}</pre>
+
+(For details on how to setup and use custom gradle configurations, see the official gradle documentation.)
```
Comment (axelfontaine) [inline documentation/gradle/index.md]: Could we link to a specific page here as well to spare users the need to search?

**Comment by axelfontaine** [comment]
Looks great! Please see my one line nitpick (should be easy to fix)

File: documentation/gradle/index.md (hunk 1 above)
Comment (simschla) [reply documentation/gradle/index.md]: good idea - of course we can :)

**Comment by simschla** [comment]
Just added the link to the official gradle documentation.

**Pull request merged**